ambassador/ambassador/ir/irambassador.py

```diff
@@ -236,6 +237,14 @@ def setup(self, ir: 'IR', aconf: Config) -> bool:
             else:
                 return False
 
+        if amod and ('retry_policy' in amod):
+            self.retry_policy = IRRETRYPOLICY(ir=ir, aconf=aconf, location=self.location, **amod.retry_policy)
+
```

Comment:
Will be helpful if we can validate the retry policy here. Look at `validate_load_balancer` for reference :+1:

Reply:
i've added this to the IRRetryPolicy class and validated retry_on in setup. would that be a correct approach to this?